src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.po.ts

```diff
@@ -1,5 +1,18 @@
+import { $$ } from 'protractor';
 import { PageHelper } from '../page-helper.po';
 
 export class MonitorsPageHelper extends PageHelper {
   pages = { index: '/#/monitor' };
+
+  getStatusTable() {
```

Comment:
All 3 new methods are not specific to the monitor page, so I think they should be moved to the `page-helper.po.ts` file.